src/jvm/test/org/jetbrains/kotlinx/lincheck_test/transformation/LocalObjectsTests.kt

```diff
@@ -243,6 +243,7 @@ class UnsafeLocalObjectsTest {
  * If we hadn't such check, this test would hang due to infinite spin-loop on a local object operations with
  * no chances to detect a cycle and switch.
  */
+@Suppress("Since15")
```

Comment:
Is this suppression necessary?

Reply:
https://github.com/JetBrains/lincheck/pull/416#discussion_r1803876177